### ISSUE TITLE
Outdated information in Readme 

### ISSUE THREAD

**Issue by maxking** [01org/ikgt-manifest#5]
Customization section of the readme.txt mentions about the build script at `pre_os/build_xmon_pkg_linux.sh` which can be used to change the memory size of the xmon but I couldn't find any such scripts in the ikgt-loader repo (which I think is inside xmon/loader). Also, since the repo has basically no history, I am not exactly sure if the information is incorrect or outdated.

**Comment by kltseng** [comment]
Yes, good catch. That's probably an outdated information that was in 1.0 release and didn't get removed when 1.1 was released. 

The new location for that is probably in pre_os/starter/xmon_desc.h (see STARTER_DEFAULT_LOAD_ADDR) though I have not tried that.
